{
  "modified": "2024-10-16T14:45:23Z",
  "published": "2024-10-16T14:45:23Z",
  "schema_version": "1.5.0",
  "id": "",
  "aliases": [
    "MAL-2024-5471"
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "pillkw"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/pillkw/MAL-2024-5471.json"
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "9a2557107023960e69444d32c178d4bb3ebc5a2b7d2e69f0f708ab2d25368811",
        "import_time": "2024-10-24T00:59:34.893463926Z",
        "id": "RLUA-2024-08681",
        "modified_time": "2024-10-16T14:45:23Z"
      }
    ]
  }
}
